library/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id 'com.google.protobuf' version '0.9.1'
+}
+
+android {
+    namespace 'org.xmtp.android.library'
+    compileSdk 33
+
+    defaultConfig {
+        minSdk 28
```

Comment:
I guess this is where the Android version matters way more! But same Q as above on if it's necessary.

Reply:
I said this in my comment above but I think we can go even lower here if wanted since the. Higher versions are more material3 and compose issues if I remember correctly.